### ISSUE TITLE
Error: params.forEach is not a function

### ISSUE THREAD

**Issue by markstos** [apidoc/apidoc#1278]
# Describe the bug

With the latest release, 0.53, I'm getthis when I try to parse some files:

```
error: params.forEach is not a function
```
I expect I have some syntax problem with my API doc somewhere. It would be great if `apidoc` could add error checking where this bug is triggered, and then tell me what line in what file triggered the error. We have dozens of files that haven't hard the `apidoc` parser run on them in a while, so there are many possibilities where the bad syntax could be. 

The debug output provides a hint where the better error checking could be helpful:

```
debug: TypeError: params.forEach is not a function
    at /node_modules/apidoc/lib/core/workers/api_param_title.js:74:16
    at Array.forEach (<anonymous>)
    at /node_modules/apidoc/lib/core/workers/api_param_title.js:71:27
    at Array.forEach (<anonymous>)
    at /node_modules/apidoc/lib/core/workers/api_param_title.js:66:16
    at Array.forEach (<anonymous>)
    at Object.postProcess (/node_modules/apidoc/lib/core/workers/api_param_title.js:65:15)
    at /node_modules/apidoc/lib/core/worker.js:105:14
    at /node_modules/lodash/lodash.js:4967:15
    at baseForOwn (/node_modules/lodash/lodash.js:3032:24)
```
I was going to try to debug by running `apidoc` against only a single file to test if particular files triggered it, but it seems `apidoc` only allows a directory name as input. 

# Information

- Version of apidoc: 0.53
- Operating system: Linux
- Node version: 16

**Comment by markstos** [comment]
I found that this error can be triggered when the group name contains a dot, which is invalid. However, this problem, the file or line number are not mentioned. It would be /very/ helpful if the group names were validated for this problem so that a helpful error message could be returned instead.

I was able to find the problem spots in a large code base by using a regular expression to search the case:

```
apiParam.*\(.*?\..*?\)
```
